packages/gcf-utils/src/gcf-utils.ts

```diff
@@ -314,7 +314,19 @@ export class GCFBootstrapper {
         // clever and instead pull up a list of repos we're installed on by
         // installation ID:
         try {
-          await this.handleScheduled(id, request, name, signature);
+          if (wrapOptions?.background) {
+            console.info('scheduling cloud task');
```

Comment:
I suspect we'll see a lot of those messages ...  any identifier we can attach to make this more useful when reading the logs?